rust/src/dcerpc/dcerpc.rs

```diff
@@ -1363,7 +1363,7 @@ fn probe(input: &[u8]) -> (bool, bool) {
 
 #[no_mangle]
 pub extern "C" fn rs_dcerpc_probe_tcp(direction: u8, input: *const u8,
-                                      len: u32, rdir: *mut u8) -> i32
+                                      len: u32, rdir: *mut u8) -> u16
```

Comment:
Should it be even `AppProto` instead of `u16` to describe it better ?

Reply:
Yes, that sounds better.